.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+node_js: 8
+git:
+  depth: false
+branches:
+  only:
+    - master
+
+dist: trusty
+sudo: required
+
```

Comment:
setup a matrix in travis.yml with two different jobs, see the MDC Web for an example.